src/Selection.js

```diff
@@ -64,6 +65,10 @@ class Selection {
     )
     this._onKeyDownListener = addEventListener('keydown', this._keyListener)
     this._onKeyUpListener = addEventListener('keyup', this._keyListener)
+    this._onDropFromOutsideListener = addEventListener(
```

Comment:
dumb question, how do you know if this is "from the outside", is it b/c for "internal" drops we don't actually use the drop/drag events?

Reply:
Yes, that's exactly right. I didn't really think about it that way but that's the reason. I think if we did move to drop events for the internals we would have to refactor this approach.